documentation/staging/content/managing-domains/accessing-the-domain/istio/istio.md

```diff
@@ -133,146 +128,35 @@ and label it for automatic injection before deploying your domain.
 ```shell
 $ kubectl create namespace domain1
 ```
+
+For a non-OpenShift environment:
+
 ```shell
 $ kubectl label namespace domain1 istio-injection=enabled
 ```
 
-##### Configuring the domain resource
+For an OpenShift environment, you do need to label the namespace for Istio sidecar injection, instead add the following
```

Comment:
Should this be 'For an OpenShift environment, you do not need to label" since you add annotation instead?

Reply:
fixed